src/app/core/signing.service.ts

```diff
@@ -0,0 +1,85 @@
+import { Injectable } from '@angular/core';
+import { Fonts } from '../shared/interfaces';
+import { ActivatedRoute, Router } from '@angular/router';
+import { AuthService } from './auth.service';
+import { MsgIoService } from './msg-io.service';
+import { FontService } from './font.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SigningService {
+  public signingBoxText;
+  public backgroundColor;
+  public textColor;
+  public canSend;
+  public charCount;
+  public maxCharCount: number;
+  public charCountColor;
+
+  // font service stuff
+  public currentFont; // Used to select a font from fontDict
+  public currentFontFamily; // Used to CSS select the font
+  public currentFontDisplay; // Shown in the Font Dropdown menu
+  public fontDict: Fonts;
+
+  constructor() {
+    this.fontDict = FontService.fonts;
+    this.maxCharCount = MsgIoService.maxMessageLength;
+  }
+
+  /**
+   * Sets all class variables to their default values
+   */
+  resetDefaultValues() {
+    this.signingBoxText = '';
+    this.backgroundColor = '#ffffff';
+    this.textColor = '#000000';
+    this.canSend = false;
+    this.charCount = 0;
+    this.charCountColor = '#bbbbbb';
+    this.currentFont = 'Arial'; // Used to select a font from fontDict
+    this.currentFontFamily = 'Arial, sans-serif'; // Used to CSS select the font
+    this.currentFontDisplay = 'Arial'; // Shown in the Font Dropdown menu
```

Comment:
```suggestion
    this.currentFont = 'Arial'; 
    this.currentFontFamily = 'Arial, sans-serif'; 
    this.currentFontDisplay = 'Arial'; 
```
Remove redundant comments (already mentioned in definition of these variables)